tools/run_tests/run_tests.py

```diff
@@ -810,7 +810,7 @@ def git(cmd, cwd=cwd):
   else:
     language_make_options = next(iter(languages)).make_options()
 
-if len(languages) != 1 or len(build_configs) != 1:
+if len(languages) != 1 and len(build_configs) != 1:
```

Comment:
well, that's tough. This condition is here for a reason. I've split up testing docker images per-language a while ago and currently, run_tests.py can only use one docker image in a single run (and yeah, I forgot about code coverage test that requires a single umbrella image when doing the change - my bad).

This check is here to protect us against cases where we want to run more languages at a time (-l all), but that would require using more docker images which we currently don't support.

I'd be actually surprised if the coverage run would actually work with this "fix".